src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,9 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static void main(String[] args) {
+
+        new FruitReportFile().report("10.09.99.vcs");
```

Comment:
In this task you have to create a few independent services and call them from main method consistently. For example you need to create service, that only reads data from file and return list of file rows  `List<String>` for example (you can call some `FileDataValidator` there to validate data). Then you can pass that `List<String>` to another service that will add/remove/... your fruits to/from some `Storage`(You need to implement some `Storage` class, where you will store fruits). Also you have to create some `dao` laver that will perform some `CRUD` operations with your storage, and call them from your `Services`.